### PR TITLE
dbaas: add new guide about prometheus endpoints for database services

### PR DESCRIPTION
**Pull request by rgbj**

### CODE REVIEW

File: pages/public_cloud/public_cloud_databases/databases_17_metrics_via_prometheus/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+> @api {v1} /cloud GET /cloud/project/{serviceName}/database/{engine}/{clusterId}/prometheus
+>
+
+Note that in this case you won't get the password; You'll need to reset the credentials if you need it.
+
+### Configure your prometheus server and start pulling metrics
+
+Use the data from the previous state to configure your prometheus server and start pulling metrics for your service 
+
+## limitations
```
Comment (thild42) [inline pages/public_cloud/public_cloud_databases/databases_17_metrics_via_prometheus/guide.en-gb.md]: L to capitalize

File: pages/public_cloud/public_cloud_databases/databases_17_metrics_via_prometheus/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+title: Public Cloud Databases - How to fetch service metrics with Prometheus
+excerpt: Find out how to setup a prometheus exporter for your Public Cloud databases service
+updated: 2024-11-25
+---
+
+## Objective
+
+Public Cloud databases allow you to export service metrics using Prometheus.
+
+**This guide explains how to setup a prometheus ex your Public Cloud databases service so that Prometheus can collect metrics.**
```
Comment (thild42) [inline pages/public_cloud/public_cloud_databases/databases_17_metrics_via_prometheus/guide.en-gb.md]: I think this sentence miss a few words.

File: pages/public_cloud/public_cloud_databases/databases_17_metrics_via_prometheus/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  scheme : https
+  basic_auth:
+    username: prometheus
+    password: "xxx"
+  dns_sd_configs:
+    - names:
+      - '_prometheus._tcp.mongodb-7bfb5d40-oa85dff3e.database.cloud.ovh.net' 
+```
+
+The aiven-backed engines use a slightly different format:
```
Comment (BellionBastien) [inline pages/public_cloud/public_cloud_databases/databases_17_metrics_via_prometheus/guide.en-gb.md]: "Aiven-backed engines" is not comprehensible for most of our customer, most of them are not aware of a third party involved in the delivery of their service. We may prefer listing impacted engines.

**Review by Y0Coss — APPROVED** [review]

**Comment by Y0Coss** [comment]
Thank you for your contribution @rgbj

**Pull request merged**